src/com/lftechnology/java/training/sanish/application/model/service/CrudService.java

```diff
@@ -0,0 +1,62 @@
+package com.lftechnology.java.training.sanish.application.model.service;
+
+import com.lftechnology.java.training.sanish.application.model.Model;
+
+import java.sql.ResultSet;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.sql.ResultSet'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)